app/queues/dns/dns-flow.server.ts

```diff
@@ -157,13 +156,13 @@ export const updateDnsRequest = async ({ id, username, type, name, value }: JobR
 };
 
 export const deleteDnsRequest = async ({ id, username, type, name, value }: JobRecord) => {
-  const fullRecordName = buildDomain(username, name);
+  logger.debug(process.env.AWS_ROUTE53_HOSTED_ZONE_ID);
```

Comment:
If you don't mind me asking, what does this logger do in this line?

Reply:
I was actully debugging something and forgot to remove it :) good catch